README.dev.md

```diff
@@ -1,31 +1,337 @@
+Apache Ant-based projects. There is a separate project for each distinct part of CLAW. Additionally all of these have 
+a binding to top-level project. Additionally, the common build-specific information and parameters is concentrated in 
+the file [&lt;build dir&gt;/cx2t/common/claw.properties](cx2t/claw.properties.in) , which mostly mirrors variables from
+[properties.cmake](properties.cmake). Majority of the projects include this file.
+<table>
+<thead>
+  <tr>
+    <th colspan="3">Projects</th>
+  </tr>
+  <tr>
+    <th>Name</th>
+    <th>Path</th>
+    <th>Description</th>
+  </tr>
+</thead>
+<tbody>
+  <tr>
+    <td>cx2t</td>
+    <td>
+
+[cx2t/src/build.xml](cx2t/src/build.xml)
+    </td>
+    <td>CLAW library</td>
+  </tr>
+  <tr>
+    <td>fetch-3rdparty-dependencies</td>
+    <td>
+
+[cx2t/common-targets.xml](cx2t/common-targets.xml)
+    </td>
+    <td>Resolves dependencies on 3rdparty libraries</td>
+  </tr>
+  <tr>
+<td>claw-unit-tests</td>
+    <td>
+
+[cx2t/unittest/build.xml](cx2t/unittest/build.xml)
+    </td>
+    <td>CLAW unit tests</td>
+  </tr>
+  <tr>
+    <td>clawfc</td>
+    <td>
+
+[driver/src/build.xml](driver/src/build.xml)
+    </td>
+    <td>CLAW driver</td>
+  </tr>
+  <tr>
+    <td>clawfc-tests</td>
+    <td>
+
+[driver/tests/build.xml](driver/tests/build.xml)
+    </td>
+    <td>CLAW driver tests</td>
+  </tr>
+  <tr>
+    <td>clawfc-unit-tests</td>
+    <td>
+
+[driver/unittests/build.xml](driver/unittests/build.xml)
+    </td>
+    <td>CLAW driver unit tests</td>
+  </tr>
+  <tr>
+    <td>claw-tests</td>
+    <td>
+
+[tests_runner/build.xml](tests_runner/build.xml)
+    </td>
+    <td>CLAW tests</td>
+  </tr>
+  <tr>
+    <td>solution</td>
+    <td>
+
+[ant-solution.xml](ant-solution.xml)
+    </td>
+    <td>Top-level project</td>
+  </tr>
+</tbody>
+</table>
+
+##IDE development
+Eclipse Is fully supported, tested with version 4.14. Each Ant project has a corresponding Eclipse project. They can be 
+imported into IDE from <build_dir>/ide/eclipse. Builds started from IDE will use the Ant build system projects 
+internally. Note that, it always uses Java installation that was specified during configuration with CMake, which does 
+not necessarily match the Java installation used by Eclipse itself. This may lead to problems during debugging, so it is
+recommended to ensure that the versions match. This can be set centrally in Eclipse settings for all the projects or 
+individually in “project/properties/java compiler/Enable Project specific settings/JDK compliance/use compliance 
+from execution environment…”.
+
+It should be further noted that these projects are not generated by CMake, but rather are produced by configuring 
+template files with CMake variables. As a consequence, each change in the projects structure and dependencies will 
+require manual adjustments of the template files. These can be found in [resources/eclipse](resources/eclipse). 
+
+#Java Coding style
```

Comment:
There needs to be a RFC to change the coding guidelines. I don't recall any discussion about changing this. 

@lxavier

Reply:
We've discussed this briefly during the last meeting. It was suggested to me that I should update the coding guidelines. 
Anyway, given that the current user base is small, I suggest that you make a list of things you want to discuss and then arrange a meeting with Xavier, so we could talk them over. This will go much faster, then having discussions here in github.